contrib/cirrus/mac_runner.sh

```diff
@@ -29,4 +29,22 @@ function logformatter() {
     "${CIRRUS_WORKING_DIR}/${SCRIPT_BASE}/logformatter" "$(output_name)"
 }
 
+# Defined by .cirrus.yml
+# shellcheck disable=SC2154
+case "$TEST_FLAVOR" in
+    mac-applehv)
+        # FIXME: Test-modes should be definitively set and positively asserted.
+        # Otherwise if the var. goes out-of-scope, defaults change, or definition
+        # fails: Suddenly assumed behavior != actual behaviorr, esp. if/when only
+        # quickly glancing at a green status check-mark.
+        export CONTAINERS_MACHINE_PROVIDER=""
```

Comment:
It makes me very uncomfortable to have an envariable set to empty. More so when the previous behavior was to have it set to `applehv`. Is it possible to make it `applehv` here? I see that it's consumed in `pkg/machine/provider/*` but haven't chased donw how it handles "" vs "actualvalue"

Reply:
Yes, makes me uncomfortable as well.  Unfortunately I literally copy/pasted most of that `FIXME` from our windows setup, which does the exact same thing :disappointed: 

Thanks for checking the code.  I'll try setting it to `applehv`, though undocumented, maybe it will 'just work'?

Reply:
Trolled through the code a bit.  It appears the default comes from containers-common.  Setting "applehv" or "libkrun" overrides it, then the value is checked against a list of static strings to catch bad names.  So this should be good (and I'll give it a try).

Reply:
[This works (:sweat:) thankfully](https://api.cirrus-ci.com/v1/artifact/task/6149203389317120/html/mac-applehv-podman-darwin-rootless-host-sqlite.log.html).  Perhaps it will inspire a similar fix for Windows testing :grin: